### DISCUSSION TITLE
Merge cellssss

### DISCUSSION THREAD

**Discussion by RTagliento** [fidley/falv#76] [Ideas]
Hi and thank you!

I love FALV

i Wish FALV integrated with this:
https://github.com/tricktresor/alv-grid-merge-cells

I Will try to do something myself.

Regards Roberto.

**Comment by fidley** [comment]
Hi Roberto,

You're more than welcome to make a contribution here!

Cheers
Łukasz

**Reply by RTagliento** [reply]
Hi,
  I just ADDED without any problem the 5 method of ZTRCKTRSR_GUI_ALV_GRID_MERGE  to ZCL_FAVL

Z_DISPLAY	Instance Method	Public
Z_INIT_CELL_STYLES	Instance Method	Public
Z_SET_CELL_STYLE	Instance Method	Public
Z_SET_FIXED_COL_ROW	Instance Method	Public
Z_SET_MERGE_HORIZ	Instance Method	Public
Z_SET_MERGE_VERT	Instance Method	Public

it works without any problem.

**Reply by fidley** [reply]
Perfect. Can you make a pull request with it? I'll will check it once I'll come back from my holidays ( next week)

**Reply by RTagliento** [reply]
I never done a pull😅.... I Will try.

**Reply by RTagliento** [reply]
Done! But a lot of differences come from different format (pretty printer) :/
